code/ATMOSPHERICS/hvac/spaceheater.dm

```diff
@@ -30,6 +54,12 @@
 	update_icon()
 	return
 
+/obj/machinery/space_heater/campfire/stove/New()
+	..()
+	cell.charge = 0
```

Comment:
Don't do that. Instead, use a variable in the space heater's New() proc to make sure a cell doesn't spawn (and that you don't use cells in general)